src/webapp/logic/utils.ts

```diff
@@ -9,3 +12,28 @@ export function getObjectVersion(object: D2ObjectWithAttributes): string | null
     );
     return versionAttributeValue && versionAttributeValue.value ? versionAttributeValue.value.trim() : null;
 }
+
+export async function getCurrentUser(api: D2Api) {
+    const authorities = await api.get<string[]>("/me/authorization").getData();
+
+    const d2CurrentUser = await api.currentUser
```

Comment:
We have already a `D2UsersRepository.getCurrentUser`, it should be used (through a use case, not directly from the react component)